### PR TITLE
Feature/quick facts integration

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR branches from feature/quick-facts-ui 

- Add integration with the quick-facts endpoint

image

### CODE REVIEW

**Comment by sorodrigo** [comment]
I tried fixing the conflicts but failed miserably. Could you have a look at them?

**Comment by Bluesmile82** [comment]
Merged develop! Please take a quick look before approving/merging

File: frontend/scripts/react-components/explore/explore.scss [hunk 1]
```diff
@@ -74,13 +75,10 @@ $explore-heading: 66px;
     .bubble {
       width: 155px;
       height: 155px;
-      display: flex;
-      flex-direction: column;
-      align-content: center;
-      justify-content: center;
+      padding-top: 55px;
```
Comment (sorodrigo) [inline frontend/scripts/react-components/explore/explore.scss]: I think this was supposed to be removed at the merge.

**Comment by Bluesmile82** [comment]
Ready!

**Pull request merged**